src/highrl/envs/teacher_env.py

```diff
@@ -98,6 +99,13 @@ def __init__(
                     "robot_num_successes",  # robot num_successes in this teacher session
                 ]
             )
+        self.teacher_reward_writer = SummaryWriter(log_dir="runs")
+        self.robot_num_successes_writer = SummaryWriter(log_dir="runs")
+        self.robot_avg_reward_writer = SummaryWriter(log_dir="runs")
```

Comment:
I think this is the same like the one in the robot_env file

Reply:
I do not understand your comment